### PR TITLE
Vault imp

### PR DESCRIPTION
**Pull request by AlbyGiaco**

### CODE REVIEW

File: frontend/src/components/vault/VaultView/VaultView.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <InfoSection />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+/*
+
```
Comment (issanyo) [inline frontend/src/components/vault/VaultView/VaultView.js:67]: why you are commenting this part?

File: frontend/src/helpers/connector.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ import { useState, useEffect } from "react";
 export const MetaMask = new InjectedConnector({ supportedNetworks: [1, 3, 4, 5, 42, 421611] });
 
 const contracts = {
-  1337: { vault: "0x2c15A315610Bfa5248E4CbCbd693320e9D8E03Cc" }, //mainnet-fork
-  3: { vault: "0x3047b2b49f104f38f3b3f1ac9b8df1c62726251e" }, // Ropsten
+  1337: { vault: "0x94f810D5F23F08e772A8d92Dd20f7561d4D6E28b" }, //mainnet-fork
+  3: { vault: "0x6a209bee7225f04b78e8addcd69dda7a7efe5ec0" }, // Ropsten
```
Comment (issanyo) [inline frontend/src/helpers/connector.js:9]: this changes shouldnt be here, maybe you need to update with master?

File: frontend/src/components/vault/VaultView/VaultView.js (hunk 1 above)
Comment (AlbyGiaco) [reply frontend/src/components/vault/VaultView/VaultView.js:67]: Removed!

File: frontend/src/helpers/connector.js (hunk 2 above)
Comment (AlbyGiaco) [reply frontend/src/helpers/connector.js:9]: Idk why this change is here

**Pull request merged**